templates/category/_items.html

```diff
@@ -17,22 +17,22 @@
       </div>
 
       <div class="panel-footer">
-        {% if availability.available %}
+        {% if product.is_available %}
           {% price_range availability.price_range %}
-          {% if availability.discount %}
-            {% if availability.price_range_undiscounted.min_price != availability.price_range.min_price %}
-              <div class="product-list__sale">
-                <svg data-src="{% static "images/sale-bg.svg" %}" />
-                <span class="product-list__sale__text">
-                  {% comment %}Translators: Layout may break if character length is different than four.{% endcomment %}
-                  {% trans "Sale" context "Sale (discount) label for item in product list" %}
-                </span>
-              </div>
-            {% endif %}
-          {% endif %}
         {% else %}
           &nbsp;
         {% endif %}
+        {% if availability.available and availability.discount %}
+          {% if availability.price_range_undiscounted.min_price != availability.price_range.min_price %}
```

Comment:
Could we somehow move these two lines to Python code? I'm afraid we'll end up with multiple slightly different implementations of the same logic.